models/refresh_token.go

```diff
@@ -40,6 +39,8 @@ func (RefreshToken) TableName() string {
 // refresh token to authenticated users.
 type GrantParams struct {
 	FactorID *uuid.UUID
+
+	SessionNotAfter time.Time
```

Comment:
should this be a pointer instead? will make it easier to check for nulls with `nil` 
```suggestion
	SessionNotAfter *time.Time
```

Reply:
Done.